src/rust/ide/view/graph-editor/src/component/visualization/metadata.rs

```diff
@@ -0,0 +1,12 @@
+//! Visualization Metadata contains information about the runtime state of the visualization.
+
+use crate::prelude::*;
+
+use crate::data::enso;
+
+/// Description of the visualization state in the IDE.
```

Comment:
Could you add description here what it is used for with a concrete example? I mean, reading just these docs I cant really understand what this struct is meant for and what use cases it is needed for. An example use case of the bug that it fixes now would be perfect.